sys/include/net/ng_pkt.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (C) 2014 Martine Lenders <mlenders@inf.fu-berlin.de>
```

Comment:
Both, the first and last year of the copyright "application" are important to determine since, and until when it holds.